py3status/parse_config.py

```diff
@@ -563,6 +563,8 @@ def parse_config(config):
         general_defaults.update(config_info['general'])
     config['general'] = general_defaults
 
+    if 'py3status' in config_info:
+        config['py3status'] = config_info['py3status']
```

Comment:
How about just

`config['py3status'] = config_info.get('py3status', {})`

that way config['py3status'] will always exist and be a dict